warehouse/models/staging/stg_waste_emission.sql

```diff
@@ -0,0 +1,227 @@
+{% set cnd_split = 0.5 %}
+{% set residual = 0.22 %}
+{% set hard_split = 0.1 %}
+
+WITH ef AS (
```

Comment:
this selects data from the waste_emission_factors_EPA source table, specifically for certain materials. It assigns a group_id based on the material type ( to help join with other data?)